packages/gatsby/src/utils/webpack-error-parser.js

```diff
@@ -0,0 +1,36 @@
+const stageCodeToReadableLabel = {
+  "build-javascript": `Generating JavaScript bundles`,
+  "build-html": `Generating SSR bundle`,
+}
+
+const transformWebpackError = (stage, webpackError) => {
+  return {
+    id: `98123`,
+    filePath: webpackError?.module?.resource,
```

Comment:
is this syntax supported? 😱

Reply:
It is - we use optional chaining babel plugin ( https://github.com/gatsbyjs/gatsby/blob/master/packages/babel-preset-gatsby-package/index.js#L44 ) for our packages.

I'm not gonna lie - VS code do complain about this, but this is so much more compact that expanding this stuff manually

And sometime we do use this ( https://github.com/gatsbyjs/gatsby/blob/35f93a041f226e04d414c343fba24a179d843dc4/packages/gatsby/src/query/query-watcher.js#L113 )

Reply:
nice, I didn't know.